src/main/java/BalancedString.java

```diff
@@ -0,0 +1,52 @@
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Stack;
+
+/**
+ * Return true if the string is balanced.
+ * example: [({})]
+ * return: true
+*/
+public class BalancedString {
+    //O(n2)
+    public static boolean isBalancedOptionOne(String s){
+        Stack<Character> c = new Stack<Character>();
+        for (int i = 0; i < s.length(); i++){
+            if(s.charAt(i) == '[' || s.charAt(i) == '(' || s.charAt(i) == '{') {
```

Comment:
use a local variable, like currentCharacter, instead of calling charAt multiple times